coalahtml/_coalahtml/app/styles/files.css

```diff
@@ -1,7 +1,20 @@
+.highlight-line {
+    background: #9E5660 !important;
```

Comment:
I think `!important` isn't considered a best practice. Can you do this without it?